src/Mod/TechDraw/Gui/CommandExtensionPack.cpp

```diff
@@ -1593,8 +1593,8 @@ CmdTechDrawExtensionExtendLine::CmdTechDrawExtensionExtendLine()
 {
     sAppModule = "TechDraw";
     sGroup = QT_TR_NOOP("TechDraw");
-    sMenuText = QT_TR_NOOP("Extend Line");
-    sToolTipText = QT_TR_NOOP("Extend a cosmetic line or centerline at both ends:<br>\
+    sMenuText = QT_TR_NOOP("Extend line");
+    sToolTipText = QT_TR_NOOP("Extends a selected cosmetic line or centerline at both ends:<br>\
```

Comment:
```suggestion
    sToolTipText = QT_TR_NOOP("Extends a selected cosmetic line or centerline at both ends by the specified delta dimenson")
```
then remove next 3  lines